frontend/app/components/Arrows.js

```diff
@@ -0,0 +1,216 @@
+                toValue: 1,
+                duration: 2000,
+                useNativeDriver: true,
+            }),
+            Animated.timing(animatedValue2, {
+                toValue: 1,
+                duration: 2000,
+                useNativeDriver: true,
+                delay: 200,
+            }),
+            Animated.timing(animatedValue3, {
+                toValue: 1,
+                duration: 2000,
+                useNativeDriver: true,
+                delay: 400,
+            }),
+        ])
+    ).start();
+
+    const animatedStyle = {
+        opacity: animatedValue.interpolate({
+            inputRange: [0, 0.5],
+            outputRange: [1, 0], // Reverse the output range
+        }),
+        transform: [
+            {
+                translateY: animatedValue.interpolate({
+                    inputRange: [0, 1],
+                    outputRange: [0, -50],
+                }),
+            },
+        ],
+    };
+
+    const animatedStyle2 = {
+        opacity: animatedValue2.interpolate({
+            inputRange: [0, 0.7],
+            outputRange: [1, 0], // Reverse the output range
+        }),
+        transform: [
+            {
+                translateY: animatedValue2.interpolate({
+                    inputRange: [0, 1],
+                    outputRange: [0, -85],
+                }),
+            },
+        ],
+    };
+
+    const animatedStyle3 = {
+        opacity: animatedValue3.interpolate({
+            inputRange: [0, 0.9],
+            outputRange: [1, 0], // Reverse the output range
+        }),
+        transform: [
+            {
+                translateY: animatedValue3.interpolate({
+                    inputRange: [0, 1],
+                    outputRange: [0, -120],
+                }),
+            },
+        ],
+    };
+
+    let pusherFunction = async () => {
+        await pusher.connect();
+        const channel = await pusher.subscribe('action');
+        channel.bind('test_event', function (data) {
+            if (typeof data === 'string') {
+                data = JSON.parse(data);
+            }
+            Object.keys(data.Actions).forEach((key) => {
+                if (JSON.parse(busId) === key) {
+                    const actionValue = Object.values(data.Actions)[0];
+                    if (actionValue === 0) {
+                        setAction('Default');
+                    } else if (actionValue === 1) {
+                        setAction('Maintain Speed');
+                    } else if (actionValue === 2) {
+                        setAction('Speed Up');
+                    } else if (actionValue === 3) {
+                        setAction('Slow Down');
+                    }
+                }
+            });
+        });
+    };
+    pusherFunction();
+    return (
+        <Theme name="light">
+            <View style={styles.container}>
+                <Text style={styles.text}>{currentAction === null ? "Default" : currentAction}</Text>
```

Comment:
i think it should just write nothing (or at least something better - eller er maintain speed så slemt?)

Reply:
Hvad har du af bud? Maintain speed er misvisende da den man jo vil starte i stilstand.

Reply:
Await Action måske?